### PR TITLE
Switching to FC prod.

### PR DESCRIPTION
**Pull request by danqrodney**
Changing billing project prefix for local, test.

Checking for null contact email before trying to use MailChimp.

### CODE REVIEW

File: api/config/config_test.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "firecloud": {
     "debugEndpoints": false,
     "billingAccountId": "014D91-FCB792-33D2C0",
-    "billingProjectPrefix": "aou-test-free1-",
+    "billingProjectPrefix": "aou-test-f1-",
     "registeredDomainName": "all-of-us-registered-test",
```
Comment (calbach) [inline api/config/config_test.json]: Does this exist yet in FC prod?

File: api/src/main/resources/fireCloud.yaml [hunk 2]
```diff
@@ -17,7 +17,7 @@ info:
     url: http://opensource.org/licenses/BSD-3-Clause
   termsOfService: https://github.com/broadinstitute/firecloud-orchestration
 
-host: "firecloud-orchestration.dsde-dev.broadinstitute.org"
+host: "api.firecloud.org"
```
Comment (calbach) [inline api/src/main/resources/fireCloud.yaml]: What about `notebooks.yaml`? Will it work if it's not in sync with FC?

Comment (blrubenstein) [reply api/src/main/resources/fireCloud.yaml]: I believe we also do need to swap `notebooks.yaml`.

Comment (blrubenstein) [reply api/src/main/resources/fireCloud.yaml]: The path is `notebooks.firecloud.org`

Comment (calbach) [reply api/src/main/resources/fireCloud.yaml]: Also there are two hardcoded references here: ui/src/app/views/workspace/component.ts

File: api/config/config_test.json (hunk 1 above)
Comment (danqrodney) [reply api/config/config_test.json]: I created it, and added the affiliated Google Group to the ACL on the CDR version we're using.

File: api/src/main/resources/fireCloud.yaml (hunk 2 above)
Comment (danqrodney) [reply api/src/main/resources/fireCloud.yaml]: Updated.

**Comment by danqrodney** [comment]
Even if we are using the same URL for all environments normally, it would probably still be better for this to be config-driven... just in case we need to point at a dev version of Leo or something in future.

**Pull request merged**